packages/odyssey-react-mui/src/labs/AppTile.tsx

```diff
@@ -134,24 +138,50 @@ const AppTile = ({
   const tileContent = useMemo(
     () => (
       <ContentContainer>
-        <Box>
+        <Box sx={{ width: "100%" }}>
           {image && (
             <ImageContainer
               odysseyDesignTokens={odysseyDesignTokens}
               hasAction={Boolean(onActionClick)}
             >
-              {image}
+              {isLoading ? (
+                <MuiSkeleton
+                  height={APP_TILE_IMAGE_HEIGHT}
+                  width={APP_TILE_IMAGE_HEIGHT}
+                  variant="rectangular"
+                />
```

Comment:
Do we want this instead?

```suggestion
                <MuiSkeleton
                  height={APP_TILE_IMAGE_HEIGHT}
                  width={APP_TILE_IMAGE_WIDTH}
                  variant="rectangular"
                />
```